cmd/axelard/cmd/vald/config/config.go

```diff
@@ -12,7 +13,7 @@ import (
 // ValdConfig contains all necessary vald configurations
 type ValdConfig struct {
 	tss.TssConfig         `mapstructure:",squash"`
-	BroadcastConfig       `mapstructure:",squash"`
+	BroadcastConfig       `mapstructure:"broadcast"`
```

Comment:
Ah, I see...
What about the TSS config, should we do the same?

Reply:
yeah that would make sense I think, as far as I can tell the config is only set with flags, so it hasn't been an issue yet